src/app/shared/interceptors/local-backend.interceptor.ts

```diff
@@ -0,0 +1,36 @@
+import { Injectable } from '@angular/core';
+import {
+  HttpRequest,
+  HttpHandler,
+  HttpEvent,
+  HttpInterceptor
+} from '@angular/common/http';
+import { delay, finalize, Observable } from 'rxjs';
+import { environment } from '../../../environments/environment';
+import { LoadingService } from '@shared/services/loading.service';
+
+@Injectable()
+export class LocalBackendInterceptor implements HttpInterceptor {
+
+  private count = 0;
+  constructor(
+    private loadingService: LoadingService
+  ) {}
+
+  intercept(request: HttpRequest<unknown>, next: HttpHandler): Observable<HttpEvent<unknown>> {
```

Comment:
We need to be careful with such solutions, because we don't really want to show loader on any http call. Some data could be updated/loaded in background without visual notification for user. Because having loader on any action could reduce UX

Reply:
There were no description in task, where should i actually attach my loader. But I got you, Can I use your solution in the real project. I think for now it is ok

Reply:
Yes, I think you will be able to change it in NgRx HW